requirements.txt

```diff
@@ -1,7 +1,8 @@
 wheel>=0.31.1
 numpy>=1.6.0
 scipy>=0.9.0
-boto3>=1.2.3
+botocore=1.19
```

Comment:
I think we would rather have an inequality (e.g. `botocore<=1.19`) than an equality, to have more flexibility on which version can be used.